src/ontology/mondo-edit.obo

```diff
@@ -157697,14 +157697,15 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0007727
-name: autosomal dominant familial periodic fever
+name: TNF receptor 1-associated periodic fever syndrome
 def: "A periodic fever syndrome, characterized by recurrent fever, arthralgia, myalgia and tender skin lesions lasting for 1 to 3 weeks, associated with skin, joint, ocular and serosal inflammation and complicated by secondary amyloidosis." [https://orcid.org/0000-0001-5208-3432, Orphanet:32960]
 subset: gard_rare {source="GARD:8457", source="MONDO:GARD"}
 subset: nord_rare {source="NORD:1804", source="MONDO:NORD"}
 subset: ordo_disorder {source="Orphanet:32960"}
 subset: orphanet_rare {source="Orphanet:32960"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "autosomal dominant familial periodic fever" BROAD []
```

Comment:
We need a x-ref for this. Maybe you can add Ada's orcid : https://orcid.org/0000-0002-1780-5230